FactoryDesignPattern.java

```diff
@@ -0,0 +1,97 @@
+import java.util.Scanner;
+
+public class FactoryDesignPattern {
+    public static void main(String[] args) {
+        Logistics l= new Logistics();
```

Comment:
use a code formatter plugin to format and beautify code.
for [vscode](https://marketplace.visualstudio.com/items?itemName=redhat.java)

Reply:
done